### ISSUE TITLE
About meta test fine tuning process

### ISSUE THREAD

**Issue by 17011813** [YuhaoWong0103/fsl_ts#2]
Hello there, thanks for your answerings.

While running and studying your code last week, I have one more question in meta test part.

https://github.com/YuhaoWong0103/fsl_ts/blob/6a5fe9fb8376c935e7415b96ead499906695fe4a/maml_higher_learner.py#L106

In the fine-tuning part in the meta test process, I want to make sure what's the **update_step_test**.
Does **update_step_test** only apply to _the support set_ right?
Then we can train and update the _test support set_ 10 times and save **the loss**.
And then we use  **this loss** with the _test query set_ for the real test?

Also, Is it okay to update the _test query set_ more than 1 time? like in original MAML code. (https://github.com/cbfinn/maml/blob/master/main.py)

Am I understanding correctly? I want to make clear about the testing process :)
Can you explain more specifically about this issue? 
Can you recommend some documents or papers that describe the meta testing process you implemented in the code in detail?

Thank you for reading my questions. Have a great day!
